pkg/generic/httpthrift_codec.go

```diff
@@ -46,15 +47,30 @@ type HTTPRequest = descriptor.HTTPRequest
 type HTTPResponse = descriptor.HTTPResponse
 
 type httpThriftCodec struct {
-	svcDsc           atomic.Value // *idl
-	provider         DescriptorProvider
-	codec            remote.PayloadCodec
-	binaryWithBase64 bool
+	svcDsc                   atomic.Value // *idl
+	provider                 DescriptorProvider
+	codec                    remote.PayloadCodec
+	binaryWithBase64         bool
+	dyConvOpts               conv.Options
```

Comment:
Pls add detailed comments for these new fields, expecially for those with abbriviated names

Reply:
btw, `dy` and `t2j` are really bad-smell names

Reply:
fixed